README.md

```diff
@@ -54,6 +54,23 @@ For these tasks you may prefer using webpki in combination with libraries like
 Changelog
 =========
 
+* Next release:
+  - *Breaking change*: added `EndEntity::verify_is_valid_tls_client_cert`
+    argument for certificate revocation lists.
+  - Added certificate revocation list (CRL) support.
+  - *Breaking change*: removed deprecated `Time::try_from` fn.
+  - *Breaking change*: removed deprecated `DnsName::From<DnsNameRef<'_>>` trait.
```

Comment:
I find this way of describing trait impls confusing (and you wrote "removed .. trait", which is arguably not what we did). I would just write "`impl From<DnsNameRef<'_>> for DnsName`" or "`From<DnsNameRef<'_>` impl for `DnsName`". (Same for the line below.)

Reply:
PTAL at the latest revision. I hope I interpreted your comment correctly.